tests/test_utils/tests_helper.py

```diff
@@ -442,6 +455,28 @@ def dump_multiple_classification(
         comparable_outputs=comparable_outputs,
     )
 
+    X, y = make_classification(40, n_features=4, random_state=42,
+                               n_classes=3, n_clusters_per_class=1)
+    X = X[:, :2]
+    model.fit(X, y)
+    if verbose:
+        print("[dump_multiple_classification] model '{}'".format(
+            model.__class__.__name__))
+    model_onnx, prefix = convert_model(model, "multi-class classifier",
+                                       [("input", FloatTensorType([1, 2]))])
+    if verbose:
+        print("[dump_multiple_classification] model was converted")
+    dump_data_and_model(
+        X.astype(numpy.float32),
+        model,
+        model_onnx,
+        folder=folder,
+        allow_failure=allow_failure,
+        basename=prefix + "Mcl" + model.__class__.__name__ + suffix + 'Rnd',
```

Comment:
Instead of so many concatenations, f-strings would be better.

Reply:
See below.